dev/teamcity/dist-assets-tc

```diff
@@ -4,6 +4,9 @@ set -o xtrace
 set -o nounset
 set -o errexit
 
+. ~/.nvm/nvm.sh
+nvm install
```

Comment:
what is this `nvm install` for?

Reply:
will install the version specified in `nvmrc` then `use` it. if it's already installed just moves straight to `use`